src/Pages/ReferenceTokens.razor

```diff
@@ -16,11 +17,17 @@
     <TabPane TabId="general" TabIndex="0" IsActive="true">
         <div class="row">
             <div>
-                <QuickGrid Items="@persistedGrants" @ref="grid" class="table table-hover table-striped" Pagination="@_pagination" >
-                    <PropertyColumn Property="c => c.ClientId" Title="Client Id" Sortable="true"/>
+                <QuickGrid Items="@persistedGrants" @ref="grid" class="table table-hover table-striped" Pagination="@_pagination">
+                    <TemplateColumn Title="Client Id"  Context="token" SortBy="@sortClientId">
+                        @{
+                            var id = configurationDbContext.Clients.Where(c => c.ClientId == token.ClientId).Select(c => c.Id).First();
```

Comment:
I'm a bit worried about the performance of this construction. Now we're querying the database every time this table renders for each row in the table. Could there perhaps be a way to query the data once per render?

Reply:
Ive resolved it i 8e9535f